test/edu/stanford/dlss/was/TestWasapiConnection.java

```diff
@@ -0,0 +1,17 @@
+package edu.stanford.dlss.was;
+
+import java.io.IOException;
+
+import org.junit.*;
+import static org.junit.Assert.*;
+import org.mockito.*;
+import org.mockito.Mockito.*;
+import org.mockito.Matchers.*;
+
+public class TestWasapiConnection {
+
+  @Test
+  public void testJsonQuerySuccess() throws IOException {
+    // todo
+  }
+}
```

Comment:
If we don't have any tests yet, I'm inclined to leave this class out altogether.